.github/workflows/modmesh_install.yml

```diff
@@ -68,7 +68,7 @@ jobs:
     - name: build_ext
       run: |
         python3 setup.py build_ext \
-          --cmake-args="-DCMAKE_BUILD_TYPE=Release -DPYTHON_EXECUTABLE=$(which python3)" \
+          --cmake-args=" --tidy='/usr/local/bin/clang-tidy;-warnings-as-errors=*;--extra-arg-before=--driver-mode=g++' -DCMAKE_BUILD_TYPE=Release -DPYTHON_EXECUTABLE=$(which python3)" \
```

Comment:
Why don't you use `.clang-tidy` file instead?

Reply:
I am just trying. I will delete it later.

Reply:
It's better to test in your own branch without updating the PR.  It's confusing.